### PR TITLE
Adding in the "add VPN route" here directly in the function

### PR DESCRIPTION
**Pull request by wmmatt**
Just looking for syntax / general code review

### CODE REVIEW

File: Function.New-ClientVPNConnection.ps1 [hunk 1]
```diff
@@ -164,13 +169,27 @@ Function New-ClientVPNConnection {
...
+            $output += "Successfully added the route [$route] to [$vpnName]"
+        }
+    } Catch {
+        $output += "Failed to add the route [$route] to [$vpnName]" 
+    }
+
+
+    $output += "Full error output for troubleshooting: $Error"
+    Invoke-Output $output
+    Break
```
Comment (sethbrasile) [inline Function.New-ClientVPNConnection.ps1:194]: `Break` isn't needed here

File: Function.New-ClientVPNConnection.ps1 [hunk 2]
```diff
@@ -76,7 +76,7 @@ Function New-ClientVPNConnection {
             HelpMessage='Choose the Authentication Method.'
         )]
         [ValidateSet('Chap','Eap','MachineCertificate','MSChapv2','Pap')]
-        [string]$AuthenticationMethod
+        [array]$AuthenticationMethod
```
Comment (sethbrasile) [inline Function.New-ClientVPNConnection.ps1:79]: Why `array`? Does `(Get-VpnConnection -AllUserConnection -Name $vpnName -EA 0).AuthenticationMethod` make it an `array`?

**Review by sethbrasile — APPROVED** [review]
Made some comments but looks good as long as you have a reason for the array thing

File: Function.New-ClientVPNConnection.ps1 (hunk 2 above)
Comment (wmmatt) [reply Function.New-ClientVPNConnection.ps1:79]: No so I'm setting multiple authentication methods by default, and just in general combining multiple authentication methods is acceptable here. For that reason I chose array.

File: Function.New-ClientVPNConnection.ps1 (hunk 1 above)
Comment (wmmatt) [reply Function.New-ClientVPNConnection.ps1]: This is done

**Review by sethbrasile — APPROVED** [review]
LGTM

**Pull request merged**